Formula/d/direwolf.rb

```diff
@@ -0,0 +1,43 @@
+class Direwolf < Formula
+  desc "Software \"soundcard\" AX.25 packet modem/TNC and APRS encoder/decoder"
+  homepage "https://github.com/wb2osz/direwolf"
+  url "https://github.com/wb2osz/direwolf/archive/refs/tags/1.7.tar.gz"
+  sha256 "6301f6a43e5db9ef754765875592a58933f6b78585e9272afc850acf7c5914be"
+  license "GPL-2.0-only"
+  head "https://github.com/wb2osz/direwolf.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "gpsd"
+  depends_on "hamlib"
+  # Further investigation and work on this
+  # formulae is needed to support linux builds. The upstream project
+  # provides their own mechanism for linux distribution. Brew is most
+  # valuable on macOS, where there is no other suitable package manager,
+  # so for now, restrict this formulae to macOS.
```

Comment:
it might be better to replace these code comments with some upstream issue in case the users have interest of using the formula in linux? what do you think?

Reply:
If it's ok with you, I'd prefer to leave this comment for now and flag it to the direwolf maintainers at a later time. My goal was to get an option going for macOS. The upstream project lists a variety of mechanisms for linux - yum, apt, and build from source. Macports and "try to build from source and cross your fingers" are the only macOS options right now, so I'm hoping to get this out there as a more modern option.

Reply:
Works for me, thanks!